src/vcf/validator.cpp

```diff
@@ -132,6 +132,65 @@ namespace ebi
         }
     }
 
+    bool is_compressed_file(std::istream &input,
+                            const std::string &source)
+    {
+        std::string message = "Input file is a compressed file ";
```

Comment:
"Input file should not be compressed (detected " + compression_type + " compression)"

Reply:
Should I extract `Input file should not be compressed (detected` to `inc/vcf/string_constants.hpp` ? it will be used twice . once in function to check with extensions and one in function to check with magic numbers

Reply:
just put the constant at the beginning of this file, or even better, use a function that receives the type of compression and builds the message and does the logging

Reply:
building a function looks good :grinning: